public/client/collections/Emojis.js

```diff
@@ -0,0 +1,34 @@
+var Emojis = Backbone.Collection.extend({
+
+  url: '/api/emojis/filenames',
+
+  model: Emoji,
+
+  getFilenames: function(){
+    $.ajax({
```

Comment:
not a backbone expert by any means, but is there a reason for not using the collection fetch in this case and giving it the createEmojis as a success callback?